src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/FormInspector.test.js

```diff
@@ -17,6 +17,9 @@ jest.mock('../stores/FormStore', () => jest.fn(function(resourceStore) {
     this.resourceKey = resourceStore.resourceKey;
     this.id = resourceStore.id;
     this.locale = resourceStore.locale;
+    this.data = {
```

Comment:
Can you please move that to the test where that belongs? I am sure there will be more representations of data we have to test against in the future.